hooks/hooks.py

```diff
@@ -1,28 +1,14 @@
 #!/usr/bin/env python
-from charmhelpers.contrib.ansible import AnsibleHooks
-from charmhelpers.contrib.ansible import install_ansible_support
+from ansiblecharm.runner import AnsibleHooks
```

Comment:
Breakout from charm helpers, ok. I can see where this is going. this is our pretext to maintaining a module that relates to but doesn't live in charm helpers right?